tests/functional/smoke/large-values1.sh

```diff
@@ -34,7 +34,7 @@ kvs_oparams="kvs-oparams cn_maint_disable=true"
 cmd kmt "$home" "$kvs" -s1 "$VLEN" "-i$KEYS" $kvs_oparams
 
 # verify no spill occurred
-cmd cn_metrics "$home" "$kvs" | cmd -e grep -q n.1,
+cmd cn_metrics "$home" "$kvs" | cmd -e grep n.1,
```

Comment:
Hmm, this one might actually depend on -q, would need to see the output from cn_metrics...  Maybe Gaurav knows..

Reply:
@gsramdasi

Reply:
Should be fine. I think the -q was there just to avoid polluting thetest output

Reply:
I can re-direct to /dev/null here.